content/references/http-websocket-apis/public-api-methods/clio-methods/server_info-clio.md

```diff
@@ -475,20 +477,104 @@ An example of a successful response:
+            "load_factor":1,
+            "clio_version":"1.0.2",
+            "validation_quorum":28,
+            "rippled_version":"1.9.1",
+            "validated_ledger": {
+                "age":7,
+                "hash":"4ECDEAF9E6F8B37EFDE297953168AAB42DEED1082A565639EBB2D29E047341B4",
+                "seq":73737719,
+                "base_fee_xrp":1E-5,
+                "reserve_base_xrp":1E1,
+                "reserve_inc_xrp":2E0
+            },
+            "cache": {
+                "size":15258947,
+                "is_full":true,
+                "latest_ledger_seq":73737719
+            }
+        },
+        "validated":true,
+        "status":"success"
+    },
+    "warnings": [
+        {
+            "id":2001,
+            "message":"This is a clio server. clio only serves validated data. If you want to talk to rippled, include 'ledger_index':'current' in your request"
+        }
+    ]
+}
+```
+
+<!-- MULTICODE_BLOCK_END -->
+
 The response follows the [standard format][], with a successful result containing an `info` object as its only field.
 
 The `info` object may have some arrangement of the following fields:
 
 | `Field`                             | Type            | Description          |
 |:------------------------------------|:----------------|:---------------------|
 | `complete_ledgers`                  | String          | Range expression indicating the sequence numbers of the ledger versions the local `rippled` has in its database. This may be a disjoint sequence such as `24900901-24900984,24901116-24901158`. If the server does not have any complete ledgers (for example, it recently started syncing with the network), this is the string `empty`. |
-| `rpc`                               | Object array    | Stats on each RPC handled by the Clip server since startup. |
+| `counters`                          | Object          | This is present only if the client connects to the `Clio` server over `localhost`.
+| `rpc`                               | Object array    | Stats on each RPC handled by the Clip server since startup. Since this is nested within the `counters` object, this is also present only if the client connects to the `Clio` server over `localhost`. |
 | `rpc.*.started`                     | Number          | Number of RPCs of this type that the Clio server has started processing since startup. |
 | `rpc.*.finished`                    | Number          | Number of RPCs of this type that the Clio server has finished processing since startup. |
 | `rpc.*.errored`                     | Number          | Number of RPCs of this type that have resulted in some sort of error since startup.  |
 | `rpc.*.forwarded`                   | Number          | Number of RPCs of this type that the Clio server has forwarded to a `rippled` P2P server since startup. |
 | `rpc.*.duration_us`                 | Number          | The total number of microseconds spent processing RPCs of this type since startup. | 
```

Comment:
I find that notation in the style of `rpc.*.whatever` gets hard to read or mentally parse, so the convention I've been using in the reference docs is to break these out into a separate table below, with the table description saying something like, "The `rpc` object is a map of API method names to API Stat Objects (see below)." and then below I have a sub-heading for "API Stats Object" with a table of the fields that (may) appear in each one. See [account_nfts](https://xrpl.org/account_nfts.html#response-format) for a practical example.

Reply:
Noted. Given that the information is correct and updates in this PR need to be published sooner than later, I suggest addressing this change in a separate PR.

Reply:
Sounds good. Don't forget to make a ticket to track the follow-up work so it doesn't get lost!